docs/cudf/source/user_guide/guide-to-udfs.ipynb

```diff
@@ -4,7 +4,8 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "# Overview of User Defined Functions with cuDF"
+    "Overview of User Defined Functions with cuDF\n",
+    "===================================="
```

Comment:
I think the heading and the line below need to be of equal length, or else Sphinx throws warnings. Can you fix these and/or verify that no warnings are thrown?
```suggestion
    "Overview of User Defined Functions with cuDF\n",
    "============================================"
```

Reply:
(I might be a little late commenting after Ashwin directed this PR to merge.)

Reply:
Some  of this is going to be fixed in https://github.com/rapidsai/cudf/pull/10685 anyway.